pubsub/.flake8

```diff
@@ -4,3 +4,9 @@ exclude =
   .git,
   *.pyc,
   conf.py
+
+ignore =
```

Comment:
That is because you use a different style for multi-line split of function calls than I do. Both are PEP8-compliant, and I find mine easier to read and more cut-and-paste friendly.

PEP 8 _explicitly_ allows the way I do it:

<img width="660" alt="screen shot 2017-07-20 at 12 55 08 pm" src="https://user-images.githubusercontent.com/4346/28436133-b948fd28-6d4a-11e7-93e6-0f2145be3249.png">